src/components/Home/Home.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { FunctionComponent } from "react";
+import { Button, Card, Col, Container, Row } from "react-bootstrap";
+import { useRedirect } from "../../hooks/useRedirect";
+import './Home.css'
+
+
+export const Home: FunctionComponent = () => {
+
+    const { redirect } = useRedirect();
+
+    return (
+        <Container fluid="md" className="pb-5 pt-3">
+            <h1>Requirements Requests Manager</h1>
+            <Row className="m-3">
+                <Col className="m-auto p-0" lg='9' sm='10' xs='12'>
+                    <Row className="mt-3 mb-3 mr-0">
+                        <Button className="mr-2 ml-auto" variant="primary"
+                            onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => redirect("/Requests", e)}
+                        >
+                            My Requests
+                        </Button>
+                        <Button className="ml-2" variant="primary"
+                            onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => redirect("/Requests/New", e)}
+                        >
+                            New Request
+                        </Button>
```

Comment:
I think you can greatly simplify this (I did not test/verify):
```
import { Link } from "react-router-dom";
...
<Link to="/Requests">
    <Button className="mr-2 ml-auto" variant="primary">My Requests</Button>
</Link>
<Link to="/Requests/New">
    <Button className="ml-2" variant="primary">New Request</Button>
</Link>
```
Bonus: I believe the above is more compatible with screen readers